### PR TITLE
setproctitle

### PR DESCRIPTION
**Pull request by kaplun**
- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -97,6 +97,7 @@
     'requests~=2.0,>=2.15.1',
     'timeout-decorator~=0.0,>=0.3.3',
     'Babel~=2.0,>=2.4.0',
+    'setproctitle~=1.1,>=1.1.10',
```
Comment (jacquerie) [inline setup.py]: `~=1.0`, otherwise it will pin `1.1`.

Comment (kaplun) [reply setup.py]: I will never understand/remember this bit :+1:

**Comment by jacquerie** [comment]
Sigh... this violated the 50/72 rule: https://stackoverflow.com/questions/2290016/git-commit-messages-50-72-formatting

**Comment by kaplun** [comment]
Wait a sec. I tried hard to respect it (I always do it). But this time it was a long URL. How do you break a URL?

**Pull request merged**